composer.json

```diff
@@ -69,10 +66,11 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "1.10-dev"
+            "dev-master": "1.11-dev"
```

Comment:
```suggestion
            "dev-master": "1.12-dev"
```

Reply:
I've changed it, but I wonder if this is even necessary at all. Does this alias serve any purpose?